### PR TITLE
Add role tags to user info in flex tab

### PR DESCRIPTION
**Pull request by nishimaki10**
@RocketChat/core 

ss

### CODE REVIEW

**Comment by lucasvanhalst** [comment]
Doesn't the `roleTags` function execute like 10 queries if the user has 5 roles assigned?

**Comment by nishimaki10** [comment]
@lucasvanhalst Certainly it looks like you said. However, I see [this](https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/rocketchat-ui-message/message/message.coffee#L4).
Hmmm... I will try to improve. Thanks.

**Comment by lucasvanhalst** [comment]
I guess you could use an [$in](https://docs.mongodb.com/v3.0/reference/operator/query/in/) query, like this: 
```javascript
model.find({
    _id: {
        $in: roles
    },
    description: {
        $exists: 1,
        $ne: ''
    }
}, {
    description: 1
})
```

**Comment by nishimaki10** [comment]
Yes, thanks. I finished.
I'm no using `$ne: ''`, but look like no problem. Empty role tag element does not generated.

**Comment by sampaiodiego** [comment]
Hey guys, I'm glad to see a performance concern from both of you =)
just for reference, I made [this test code](https://gist.github.com/sampaiodiego/8e559988093989a9cde1bce411cb3929) to compare the performance of the "roles tags selector" and got the following results:

  | old code (ms) | new code (ms)
----|--------|------
1 role | 851 | 1763
2 roles | 1951 | 2294
3 roles | 2833 | 2032
4 roles | 4105 | 2202

I'm looking into this PR and the from @lucasvanhalst .. hope to merge both soon :wink: 

good job guys.. thanks for the contribution

File: packages/rocketchat-ui-message/message/message.coffee [hunk 1]
```diff
@@ -5,7 +5,7 @@ Template.message.helpers
 		unless RocketChat.settings.get('UI_DisplayRoles')
 			return []
 		roles = _.union(UserRoles.findOne(this.u?._id)?.roles, RoomRoles.findOne({'u._id': this.u?._id, rid: this.rid })?.roles)
-		return _.compact(_.map(roles, (role) -> return RocketChat.models.Roles.findOne({ _id: role, description: { $exists: 1 } })?.description));
+		return RocketChat.models.Roles.find({ _id: { $in: roles }, description: { $exists: 1 } }, { description: 1, _id: 0 })
```
Comment (sampaiodiego) [inline packages/rocketchat-ui-message/message/message.coffee]: to restrict fields returned from the query, you need to specify the [fields](http://docs.meteor.com/api/collections.html#fieldspecifiers) property on the second parameter:
```
return RocketChat.models.Roles.find({ _id: { $in: roles }, description: { $exists: 1 } }, { fields: { description: 1 } })
```

**Comment by lucasvanhalst** [comment]
@sampaiodiego That's interesting.
Then again, I thought this was a database query being done because I didn't quite understand the differences between client and server, so after all it doesn't really matter I guess.

**Comment by sampaiodiego** [comment]
@lucasvanhalst it's only client side.. but I liked the improvement.. it's more consistent no matter the number of roles a user has. =)

**Pull request merged**